### ISSUE TITLE
connection_status is missing from user object

### ISSUE THREAD

**Issue by fzn0x** [twitterdev/twitter-api-typescript-sdk#89]
One line summary of the issue here.

### Expected behavior

connection_status is available

### Actual behavior

connection_status is missing from user object

### Steps to reproduce the behavior

[connection_status is missing from user object](https://devcommunity.x.com/t/announcing-connection-status-field-in-the-user-object-in-the-x-api-v2/212588)

**Comment by fm2055** [comment]
Having the same issue. "connection_status" is missing in type definition. However, endpoint actually works (though the connection_status entry is ommited in the reseponse if there is no connection between the two accounts.

**Comment by tansanDOTeth** [comment]
I'm looking for this too! Did anyone figure this out?

**Comment by fzn0x** [comment]
> I'm looking for this too! Did anyone figure this out?

@fm2055 Indeed, actually the Twitter documentation needs update too, seems like some people have no idea how `connection_status` actually works
@tansanDOTeth  I'm able to solve this by pushing a contribution in unofficial SDK https://github.com/PLhery/node-twitter-api-v2/pull/520

**Comment by fzn0x** [comment]
Maybe moving to unofficial SDK is best option for now since @PLHery actively maintain the SDK.

**Comment by tansanDOTeth** [comment]
> > I'm looking for this too! Did anyone figure this out?
> 
> @fm2055 Indeed, actually the Twitter documentation needs update too, seems like some people have no idea how `connection_status` actually works @tansanDOTeth I'm able to solve this by pushing a contribution in unofficial SDK [PLhery/node-twitter-api-v2#520](https://github.com/PLhery/node-twitter-api-v2/pull/520)

Thank you!!
